pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -116,7 +116,7 @@ func (s *Service) ReconcileLoadBalancer(openStackCluster *infrav1.OpenStackClust
 			return err
 		}
 		if pool == nil {
-			s.logger.Info("Creating load balancer pool", "name", lbPortObjectsName)
+			s.logger.Info("Creating load balancer pool", "name", lbPortObjectsName, "listenerID", listener.ID)
```

Comment:
```suggestion
			s.logger.Info(fmt.Sprintf("Creating load balancer pool to listener %q", listener.ID), "name", lbPortObjectsName, "lb-id", lb.ID)
```

Reply:
Hm we should either drop the `lb-` prefix from the lb-id or should prefix the name accordingly. Can we try to find out what the best practice is?